framework/resource/internal/wrapper.go

```diff
@@ -0,0 +1,7 @@
+package internal
```

Comment:
`internal` packages are special in Gol. In this case can be imported only by resource.